Main.java

```diff
@@ -0,0 +1,16 @@
+import shop.Customer;
+import shop.Supplier;
+
+// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
+// then press Enter. You can now see whitespace characters in your code.
+public class Main {
+    public static void main(String[] args) {
+        // Press Opt+Enter with your caret at the highlighted text to see how
+        // IntelliJ IDEA suggests fixing it.
+        Customer a = new Customer();
```

Comment:
always give meaningful name for your variables. 
```suggestion
        Customer customer = new Customer();
```